src/main/java/com/minecolonies/coremod/blocks/BlockConstructionTape.java

```diff
@@ -231,11 +231,11 @@ private void initBlock()
     @Deprecated
     public IBlockState getStateFromMeta(final int meta)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1133)